src/main/java/org/isf/utils/jobjects/JFromDateToDateChooser.java

```diff
@@ -84,12 +82,7 @@ private CustomJDateChooser getCustomJDateFrom(Date dateFrom) {
 			dateFromDateChooser.setDateFormatString(DATE_TIME_FORMAT); //$NON-NLS-1$
 			dateFromDateChooser.setFont(new Font("Arial", Font.BOLD, TEXT_SIZE), false);
 			dateFromDateChooser.setPreferredSize(new Dimension(100, 24));
-			dateFromDateChooser.addPropertyChangeListener("date", new PropertyChangeListener() {
-
-				public void propertyChange(PropertyChangeEvent evt) {
-					dateTimeFrom = dateFromDateChooser.getDate();
-				}
-			});
+			dateFromDateChooser.addPropertyChangeListener("date", evt -> dateTimeFrom = dateFromDateChooser.getDate());
```

Comment:
also `evt`..

Reply:
As noted above the variable was the one previously there: `public void propertyChange(PropertyChangeEvent evt) `

Reply:
Yes I know, I meant just to perform a find/replace